node/peer.js

```diff
@@ -205,7 +216,13 @@ TChannelPeer.prototype.countConnections = function countConnections(direction) {
 // ensures that a connection exists
 TChannelPeer.prototype.connect = function connect(outOnly) {
     var self = this;
-    var conn = self.getIdentifiedOutConnection();
+    var conn = null;
+    if (self.preferConnectionDirection === 'in' && !outOnly) {
```

Comment:
It is still used. I feel like the second is unnecessary though.
1) https://github.com/uber/tchannel/blob/master/node/hyperbahn/service_proxy.js#L279
2) https://github.com/uber/tchannel/blob/master/node/peer_score_strategies.js#L67